api/src/pcapi/routes/backoffice/offers/blueprint.py

```diff
@@ -631,3 +636,48 @@ def _batch_reject_offers(offer_ids: list[int]) -> None:
         favorites = users_models.Favorite.query.filter(users_models.Favorite.offerId.in_(offer_ids)).all()
         repository.delete(*favorites)
         search.async_index_offer_ids(offer_ids)
+
+
+@list_offers_blueprint.route("/<int:offer_id>/details", methods=["GET"])
+@utils.permission_required(perm_models.Permissions.READ_OFFERS)
+def get_offer_details(offer_id: int) -> utils.BackofficeResponse:
+    offer_query = offers_models.Offer.query.filter(offers_models.Offer.id == offer_id).options(
+        sa.orm.joinedload(offers_models.Offer.venue),
+        sa.orm.joinedload(offers_models.Offer.venue).joinedload(offerers_models.Venue.managingOfferer),
+    )
+    offer = offer_query.one_or_none()
+    if not offer:
+        flash("Cette offre n'existe pas", "warning")
+        return redirect(url_for("backoffice_web.offer.list_offers"), code=303)
```

Comment:
On n'a aucune raison d'avoir un lien vers une offre qui n'existe pas, donc on peut renvoyer une erreur 404 (on fait cela dans chaque écran). Ce commentaire est aussi valide pour le reste de la PR.
```suggestion
        raise NotFound()
```